clients/drcachesim/common/trace_entry.h

```diff
@@ -191,6 +191,35 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_CPU_ID,
 
+    /**
+     * The marker value contains function id defined by the user in droption.
+     */
+    TRACE_MARKER_TYPE_FUNC_ID,
+
+    // XXX i#3048: replace return address with callstack information.
+    /**
+     * The marker value contains the return address of the function call,
+     * whose id is specified by the closest previous
```

Comment:
My prior review comments here were not acted on: if you disagree with a review request, that's certainly your prerogative, but please reply either way to say whether you're acting on the request or why you're not -- please don't skip over it with no comment because then the reviewer wonders whether some other requests were silently skipped over...I shouldn't have to go double-check all my prior comments.

Reply:
Skipping those comments is definitely NOT deliberate... It is my bad to skip them. Because sometimes I made several commits, and a non-relevant code change might touch the lines that are covered by a comment, which will then turn this comment into hidden, which cause me to skip it. I should have been aware of that a hidden commit is not necessary addressed.
In later commit, I will manually reply all comments (including all the previous one) to ensure that they are addressed. Sorry for the inconvenience.

Reply:
Done